GTAPWiNDC/gtap9/build.gms

```diff
@@ -53,43 +53,45 @@ $label flex2gdx
 put_utility 'title' /'Reading FLEXAGG data files';
 
 loop(yr,
-	put_utility 'shell' / 'if exist ',yr.tl,'\nul rmdir /q /s ',yr.tl;
-	put_utility 'shell' / 'mkdir ',yr.tl;
+	put_utility 'shell' / 'if exist %system.fp%',yr.tl,'\nul rmdir /q /s %system.fp%',yr.tl;
+	put_utility 'shell' / 'mkdir %system.fp%',yr.tl;
 );
 
 *	Point to the directory with the flexlagg data files:
 
-$set flexagg ../../data/GTAPWiNDC/gtap9/
+$set flexagg %system.fp%../../data/GTAPWiNDC/gtap9/
 
 set		flexaggfile /
 		2004	"%flexagg%flexagg9aY04.zip"
 		2007	"%flexagg%flexagg9aY07.zip"
 		2011	"%flexagg%flexagg9aY11.zip" /;
 
 
+put_utility 'shell' / "if exist %system.fp%tmp\nul rmdir /q /s %system.fp%tmp";
```

Comment:
This is bad solution to an issue I am having. I'm trying to build this file using 

gams GTAPWiNDC/build.gms

and am getting an issue where flex2gdx is deleting the scratch directory before gdx2gdx can use it. This is not an issue if I call

gams GTAPWiNDC/gtap9/build.gms

The solution is to create a temporary directory and point both flex2gdx and gdx2gdx to this location. This is a brutish solution.

I think there are several potential solutions

1. Fix the scratch directory issue
2. Keep the current solution, but delete the temporary directory manually
3. Create a loadpoint file structure like in `core`
4. Rewrite flex2gdx so that it's part of gdx2gdx

Any suggestions @tfoxrutherford or @andrewwschreiber?